docs/guides/3_generating_proofs.md

```diff
@@ -90,4 +90,66 @@ aligned submit \
 ```
 Where proof path is the path to the proof file, public input path is the path to the public input file, and verification key path is the path to the verification key file.
 
-For more instructions on how to submit proofs, check the [Submitting proofs guide](../guides/0_submitting_proofs.md).
\ No newline at end of file
+For more instructions on how to submit proofs, check the [Submitting proofs guide](../guides/0_submitting_proofs.md).
+
+## Risc0
+
+### Dependencies
+
+This guide assumes that:
+- Risc0 toolchain installed (instructions [here](https://dev.risczero.com/api/zkvm/quickstart#1-install-the-risc-zero-toolchain))
+- Risc0 project to generate the proofs (instructions [here](https://dev.risczero.com/api/zkvm/quickstart#2-create-a-new-project))
+- Aligned installed (instructions [here](../introduction/1_getting_started.md#quickstart))
+
+### How to generate a proof
+
+First, open the risc0 host file and add the following code to export image id & public input needed by Aligned.
+
+```rust
+fn main() {
+    // your code here
+  
+    std::fs::write("<image_id_file_path>", convert(&<METHOD_ID>))
```

Comment:
lets clarify that `image_id_file_path` is the path where you want it to be written. Also, should it be image id or method id? lets unify terminology so its less confusing

Reply:
changed on ef93dfb539db69405643b476a6b8f78e4ccc5dfc